packages/kbn-cell-actions/src/actions/filter/create_filter.ts

```diff
@@ -18,21 +18,32 @@ import { DefaultActionsSupportedValue } from '../types';
 export const isEmptyFilterValue = (value: Array<string | number | boolean>) =>
   value.length === 0 || value.every((v) => v === '');
 
-const createExistsFilter = ({ key, negate }: { key: string; negate: boolean }): ExistsFilter => ({
-  meta: { key, negate, type: FILTERS.EXISTS, value: 'exists' },
+const createExistsFilter = ({
+  key,
+  negate,
+  dataViewId,
+}: {
+  key: string;
+  negate: boolean;
+  dataViewId: string | null;
+}): ExistsFilter => ({
+  meta: { key, negate, type: FILTERS.EXISTS, value: 'exists', index: dataViewId ?? undefined },
```

Comment:
Consider making `dataViewId` an optional prop instead of using `null`, it would make everything a bit easier:
```suggestion
  dataViewId?: string;
}): ExistsFilter => ({
  meta: { key, negate, type: FILTERS.EXISTS, value: 'exists', index: dataViewId },
```